packages/webapp/src/containers/Task/saga.js

```diff
@@ -339,24 +353,44 @@ export function* getTasksSuccessSaga({ payload: tasks }) {
     },
     {},
   );
+
+  const productsByTranslationKey = {};
   const tasksByTranslationKey = tasks.reduce((tasksByTranslationKey, task) => {
     const { task_translation_key } = taskTypeEntities[task.task_type_id];
     if (taskTypeActionMap[task_translation_key]) {
       tasksByTranslationKey[task_translation_key].push(task[task_translation_key.toLowerCase()]);
     }
+    if (TASKTYPE_PRODUCT_MAP[task_translation_key]) {
+      if (!productsByTranslationKey[task_translation_key]) {
+        productsByTranslationKey[task_translation_key] = [];
+      }
+      productsByTranslationKey[task_translation_key] = [
+        ...productsByTranslationKey[task_translation_key],
+        ...task[TASKTYPE_PRODUCT_MAP[task_translation_key]],
+      ];
+    }
     return tasksByTranslationKey;
   }, tasksByTranslationKeyDefault);
+
   for (const task_translation_key in taskTypeActionMap) {
     try {
       yield taskTypeActionMap[task_translation_key].success(
         tasksByTranslationKey[task_translation_key],
       );
+      yield taskProductTypeActionMap[task_translation_key]?.success(
+        productsByTranslationKey[task_translation_key],
+      );
     } catch (e) {
       yield put(taskTypeActionMap[task_translation_key].fail(e));
       console.log(e);
     }
   }
-  yield put(addManyTasksFromGetReq(tasks));
+
+  yield put(successAction(tasks));
+}
+
+export function* getTasksSuccessSaga({ payload: tasks }) {
```

Comment:
Do we need to create a new function if we still have` getAllTasksSuccessSaga()` with the same definition.

Reply:
I think I misread the diff that `getTasksSuccessSaga()` was a new function... I see the removal above now!